### PR TITLE
added logging and install script

### PR DESCRIPTION
**Pull request by sourTaste000**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,10 +17,16 @@ node index.js
...
 ```
 node >=12
+git(latest)
+go(latest)
+GNU make(latest)
 
 //on Linux
 sudo apt-get install build-essential libcairo2-dev libpango1.0-dev libjpeg-dev libgif-dev librsvg2-dev
 sudo apt-get install ffmpeg
+git clone https://github.com/buildkite/terminal-to-html.git && cd terminal-to-html
```
Comment (dominikaaaa) [inline README.md:27]: can't we have index.js do this with a shell script?

Comment (sourTaste000) [reply README.md:27]: what

Comment (sourTaste000) [reply README.md:27]: in js?

Comment (sourTaste000) [reply README.md:27]: thats not a npm package

Comment (dominikaaaa) [reply README.md:27]: yes, like the ;update command

Comment (sourTaste000) [reply README.md:27]: Like ;installsependencies
?

Comment (dominikaaaa) [reply README.md:27]: just when index first runs it'll do that. you can have a txt file or smth to see if it's run before

Comment (sourTaste000) [reply README.md:27]: Skrub be like Jason bad

Comment (dominikaaaa) [reply README.md:27]: just create a blank text file called firstRun and see if it exists or some shit

Comment (sourTaste000) [reply README.md:27]: Ye that what I was thinking... lol

Comment (humboldt123) [reply README.md:27]: Json is really really bad practice

Comment (humboldt123) [reply README.md:27]: https://www.thomascerqueus.fr/json-fields-relational-databases-postgresql/

**Comment by dominikaaaa** [comment]
I'll review this later, don't merge

**Comment by sourTaste000** [comment]
when

**Comment by dominikaaaa** [comment]
soon

**Comment by sourTaste000** [comment]
wait i gotta fix the script

**Comment by sourTaste000** [comment]
dont merge it yet

File: install.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
+if [[ $EUID -ne 0 ]]; then
+   echo "This script must be run as root"
```
Comment (dominikaaaa) [inline install.sh:2]: why??

File: install.sh [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+if test -f "firstRun"; then
+    echo "Already installed!"
+    exit 0
+fi
+
+apt-get install build-essential libcairo2-dev libpango1.0-dev libjpeg-dev libgif-dev librsvg2-dev ffmpeg
+git clone https://github.com/buildkite/terminal-to-html.git && cd terminal-to-html
+make
+mv ./terminal-to-html /usr/local/bin
```
Comment (dominikaaaa) [inline install.sh:14]: why do you need to install this system wide? just make it ~/.local/bin and make sure that's inside $PATH

File: install.sh (hunk 2 above)
Comment (sourTaste000) [reply install.sh:2]: cuz apt-get, make and mv

File: install.sh (hunk 3 above)
Comment (sourTaste000) [reply install.sh:14]: some distro doesnt have `~/.local/bin`

File: install.sh (hunk 2 above)
Comment (dominikaaaa) [reply install.sh:2]: that doesn't need to be run as sudo except for apt-get which should be done seperately just like the rest of the existing dependencies

File: install.sh (hunk 3 above)
Comment (sourTaste000) [reply install.sh:14]: nvm

Comment (dominikaaaa) [reply install.sh:14]: then you create it. It's a standard folder, which is why I said you add it to your path. Not all distros have /usr/local either

**Comment by sourTaste000** [comment]
added anime module lol

**Comment by humboldt123** [comment]
@sourTaste000 remove the anime command and have it in a different pr

**Comment by sourTaste000** [comment]
k done

**Comment by sourTaste000** [comment]
merge when

File: main.js [hunk 4]
```diff
@@ -137,6 +137,17 @@ client.on('messageReactionAdd', async (reaction, user) => {
...
+ * @module rawPastebin
+ * @author sourTaste000
+ */
+client.on('message', async message => {
+    if(message.content.includes("pastebin.com/")){
+        if (message.author.bot) return;
+        let rawpaste = message.content;
+        message.channel.send(`Raw:\n${rawpaste.replace("pastebin.com/", "pastebin.com/raw/")}`)
+    }
+})
```
Comment (zeroeightysix) [inline main.js:150]: This is cool, but does it belong in this PR?

Comment (sourTaste000) [reply main.js:150]: kill me lol

Comment (sourTaste000) [reply main.js:150]: I can't edit code on phone I will do it later

Comment (sourTaste000) [reply main.js:150]: revert*

Comment (dominikaaaa) [reply main.js:150]: it doesn't belong but I made the suggestion to do this. I'll let this one slide and merge it anyways

**Comment by dominikaaaa** [comment]
I still need to review this, don't merge

File: main.js (hunk 4 above)
Comment (sourTaste000) [reply main.js:150]: thx uwu

**Comment by sourTaste000** [comment]
</img>

**Comment by sourTaste000** [comment]
ez conflict

**Comment by sourTaste000** [comment]
moved to #133

**Pull request closed without merging**